### PR TITLE
feat: add `/networks` api

### PR DESCRIPTION
**Pull request by pdp2121**
## High Level Overview of Change

Add `\networks` endpoint to display all networks/ sidechains on VHS database. Would be useful for internal development, other API calls, and DID tooling.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: src/api/routes/v1/index.ts [hunk 1]
```diff
@@ -37,4 +38,6 @@ api.use('/network/validator/:publicKey', handleValidator)
 api.use('/network/validators/:param', handleValidators)
 api.use('/network/validators', handleValidators)
 
+api.use('/network/networks', handleNetworks)
```
Comment (mvadari) [inline src/api/routes/v1/index.ts:41]: lol

**Review by mvadari — COMMENTED** [review]
Can you add a test, since it's an easy feature to test?

**Comment by mvadari** [comment]
nit: the `/` is the wrong way in the PR title

File: src/api/routes/v1/index.ts (hunk 1 above)
Comment (pdp2121) [reply src/api/routes/v1/index.ts:41]: lol that's what I thought initially. Open to any suggestion if any

Comment (mvadari) [reply src/api/routes/v1/index.ts:41]: I don't have a better option either, it's just funny

**Review by ckniffen — APPROVED** [review]

File: src/api/routes/v1/index.ts (hunk 1 above)
Comment (ckniffen) [reply src/api/routes/v1/index.ts:41]: why cant it be just `/networks`

Comment (mvadari) [reply src/api/routes/v1/index.ts:41]: All the VHS endpoints are at `/v1/network/...`, that'd be a major change

**Review by mvadari — APPROVED** [review]

**Pull request merged**